### DISCUSSION TITLE
Default page for folder

### DISCUSSION THREAD

**Discussion by Keith-Balding** [requarks/wiki#5007] [Help / Questions] [answered]
First, apologies if this has already been covered 

I would like to have a default documents that opens when you navigate to a folder so that I can inform users what should be placed in the folder i.e. what the pages in the folder should should refer to.  I noticed that there was a previous bug report about a landing page which seems to be referring to the same functionality but there seems to be no response to that.

**Comment by NGPixel** [comment, marked as answer]
If you have a folder at `/foo-bar/...`, then you need a page `/foo-bar` to act as the landing page for that folder.
